homepage/homepage/app/docs/api-reference/jazz-browser-media-images/page.tsx

```diff
@@ -1,5 +1,10 @@
 import { PackageDocs } from "@/components/docs/packageDocs";
 
+export const metadata = {
+    title: "jazz-browser-media-images - jazz",
+    description: "API reference for jazz-browser-media-images.",
+};
+
 export default function Page() {
```

Comment:
Please may you use a parameter in the Next.js file route like `/app/docs/api-reference/[package]/page.tsx` ?

Reply:
Thanks. Done.
I forgot to mention that I also removed FAQ since it was empty.